smart_todo.gemspec

```
@@ -30,8 +30,8 @@ Gem::Specification.new do |spec|
   spec.executables   = ['smart_todo']
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency("bundler", "~> 1.17")
-  spec.add_development_dependency("rake", "~> 10.0")
+  spec.add_development_dependency("bundler", ">= 2.2.17")
```

Do we require bundler 2? I think should be sufficient
```suggestion
  spec.add_development_dependency("bundler", ">= 1.17")
```

---

I upgraded bundler here to resolve https://github.com/Shopify/smart_todo/security/dependabot/smart_todo.gemspec/bundler/open.